packages/expand-button/expand-button.scss

```diff
@@ -88,6 +88,12 @@ $expand-button-focus-color--inverted: jkl.$color-snohvit;
         }
     }
 
+    &--icon-only {
+        .jkl-expand-button__arrow {
+            margin: jkl.$spacing-2xs jkl.$spacing-xs;
```

Comment:
Jeg vurderte å legge inn en mindre margin for compact, men jeg tror det er viktig å opprettholde størrelsen på et klikkbart område.